scripts/bitcoin.sh

```
@@ -0,0 +1,67 @@
+
+  # load wallet and create it if not exists
+  info "load $wallet_name wallet"
+  bitcoin_cli createwallet $wallet_name &>/dev/null || bitcoin_cli loadwallet $wallet_name
+}
+
+bitcoin_init() {
+  docker_wait_for_healthy_service bitcoin
+
+  bitcoin_load_wallet dev
+
+  local blockcount=$(bitcoin_cli getblockcount 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)